drivers/src/nrfx_nvmc.c

```diff
@@ -415,6 +457,18 @@ void nrfx_nvmc_words_write(uint32_t addr, void const * src, uint32_t num_words)
     nvmc_readonly_mode_set();
 }
 
+uint16_t nrfx_nvmc_otp_halfword_read(uint32_t addr)
```

Comment:
This function looks a bit weird here actually. It is not related to NVMC. Perhaps it would be better to move it to `nrfx_common.h` (e.g. as `nrfx_halfword_get()`)?

Reply:
Sounds good to me. Makes the name shorter as well :). @kl-cruz what do you think?